vendor/vendor.json

```diff
@@ -367,6 +385,12 @@
 			"version": "v2.0.0-preview.5",
 			"versionExact": "v2.0.0-preview.5"
 		},
+		{
+			"checksumSHA1": "glCzRHKY3rNJua7zOZrYeHkJ3DE=",
+			"path": "github.com/aws/aws-sdk-go-v2/service",
```

Comment:
Hmmm why are we importing the whole service directory here?

Reply:
Good catch

Reply:
Just removed it!